it/it-tests/src/test/java/pageobjects/projects/FacetItem.java

```diff
@@ -38,4 +38,14 @@ public FacetItem shouldHaveValue(String key, String value) {
   public void selectValue(String key) {
     this.elt.$(".facet[data-key=\"" + key + "\"]").click();
   }
+
+  public FacetItem selectOptionItem(String value) {
+    this.elt.$(".Select-input input").val(value).pressEnter();
+    return this;
+  }
+
+  public FacetItem sortList(Integer idx) {
```

Comment:
I'd not expose the fact that the first link sorts ascending, the second one descending.

Instead let's expose two methods: `sortListAsc` and `sortListDesc`. Internally they can rely on this *private* method.